model/version.go

```diff
@@ -734,7 +734,10 @@ func constructManifest(v *Version, projectRef *ProjectRef, moduleList ModuleList
 			}
 		}
 		var sha, url string
-		owner, repo := module.GetRepoOwnerAndName()
+		owner, repo, err := thirdparty.ParseGitUrl(module.Repo)
+		if err != nil {
+			return nil, errors.Wrapf(err, "can't parse git url '%s'", module.Repo)
```

Comment:
nit: should be `"parsing git url '%s'` since that's what it was in the other diffed line.